### ISSUE TITLE
Totara Support

### ISSUE THREAD

**Issue by LiamKearn** [catalyst/moodle-tool_s3logs#31]
Do you guys use this for Totara sync logs (if you use those)?

Would a PR to support that be welcome?

**Comment by dmitriim** [comment]
I don't think we have any Totara client using this plugin. Please feel free to shoot PR to support totara.
